src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -417,11 +417,11 @@ class Foo
             {
                 SetupVerifiableGlyph(StandardGlyphGroup.GlyphGroupMethod, StandardGlyphItem.GlyphItemPrivate);
                 var item = (await _aggregator.GetItemsAsync("static")).Single();
-                VerifyNavigateToResultItem(item, "static", MatchKind.Exact, NavigateToItemKind.Method, "@static()", $"{FeaturesResources.type_space}Foo");
+                VerifyNavigateToResultItem(item, "static", MatchKind.Exact, NavigateToItemKind.Method, "static()", $"{FeaturesResources.type_space}Foo");
```

Comment:
No need to escape items we show in NavigateTo.  Plus, it doesn't match what we do elsewhere (like in nav-bars): 

![image](https://cloud.githubusercontent.com/assets/4564579/25061006/c84bd86c-2160-11e7-9356-cc55a713a595.png)

Also, if we escape them, then the highlight spans we compute won't be correct in the display name.